force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,106 @@
+<template> 
+    <lightning-card title="PD1 Calculator">
+        <lightning-button 
+            variant="brand" 
+            label="Calculate" 
+            title="Calculate" 
+            slot="actions"
+            onclick={calculateScore}
+            class="slds-size_x-small"
+            ></lightning-button>
+        <div class="slds-p-horizontal_small">
+        <!-- Input information -->  
+            <Lightning-input 
+                type="number" 
+                name="devFundamentals" 
+                label="Developer Fundementals: 23%"
+                value={devFundamentals}
+                onchange={handleChange}
+                class="slds-size_x-small"></Lightning-input>
+            <Lightning-input 
+                type="number" 
+                ame="processAutomation" 
+                label="Process Automation: 30%"
+                value={processAutomation}
+                onchange={handleChange}
+                class="slds-size_x-small"></Lightning-input>
+            <Lightning-input 
+                type="number" 
+                name="userInterface" 
+                label="User Interface: 25%<"
+                value={userInterface}
+                onchange={handleChange}
+                class="slds-size_x-small"></Lightning-input>
+            <Lightning-input 
+                type="number" 
+                name="testDebugDeploy" 
+                label="Testing, Debugging, and Deployment: 22%"
+                value={testDebugDeploy}
+                onchange={handleChange}
+                class="slds-size_x-small"></Lightning-input>
```

Comment:
Try adding some validation on the input to stop the user from adding values over 100% or under 0%